CMakeLists.txt

```diff
@@ -75,9 +75,13 @@ list(APPEND kinds "4" "8" "d")
 
 # Set common to the package compiler flags
 if(CMAKE_C_COMPILER_ID MATCHES "^(Intel)$")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -traceback")
   set(CMAKE_C_FLAGS_DEBUG "-O0")
+  set(CMAKE_C_FLAGS_RELEASE "-O3")
 elseif(CMAKE_C_COMPILER_ID MATCHES "^(GNU)$")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -fbacktrace")
```

Comment:
`-fbacktrace` is valid for Fortran and not C.

Reply:
OK, is there something comparable for the gcc compiler?

At least for Intel, `-ftraceback` is valid for both the ifort and icc compilers.

Reply:
Interestingly enough, I don't think there is.

Reply:
Thanks Rahul and Kyle!  I'll push up another commit to remove the `-fbacktrace` for the C compiler, so the only remaining flag there will just be `-g`.